src/Nethermind/Nethermind.Network/P2P/Subprotocols/Etha/EthaProtocol.cs

```diff
@@ -0,0 +1,17 @@
+using Nethermind.Core.Crypto;
+using Nethermind.Network.P2P;
+
+namespace Nethermind.Network.P2P.Subprotocols.Etha
+{
+    public class EthaProtocol : Protocol
+    {
+        public override string Name => "etha";
+        public override byte Version => 1;
+        public override int MessageIdSpaceSize => 3; // Number of messages in protocol
```

Comment:
I would put it directly to `EthaProtocolHandler` to make it similar to implementations of other protocols. `EthaProtocol` can be removed.